dependency-lint.yml

```diff
@@ -0,0 +1,46 @@
+# array of strings or regular expressions to match against a module name
+# to determine if it is allowed to be unused
+allowUnused:
+  - cucumber
+  - livescript
+
+# array of path patterns to match againt a filename
+# to determine if it is used only for development
+#   (see https://github.com/isaacs/minimatch)
+devFilePatterns:
+  - '{features,spec,test}/**/*'
+  - '**/*_{spec,test}.js'
+
+# array of strings or regular expressions to match against a script name
+# in your `package.json` to determine if it is used only for development
+devScripts:
+  - lint
+  - publish
+  - test
+  - version
+
+# path pattern to match against a filename to determine if it should be parsed.
+# This is the starting point, devFilePatterns and ignoreFilePatterns should be
+# subsets of this pattern
+#   (see https://github.com/isaacs/minimatch)
+filePattern: '**/*.ls'
+
+# array of path patterns to match against a filename
+# to determine if it should be ignored
+#   (see https://github.com/isaacs/minimatch)
+ignoreFilePatterns:
+  - 'node_modules/**/*'
```

Comment:
Mainly did this just to make it very explicit what is ignored. If its not there, then I would just be adding a comment to the explantation explaining that `node_modules` is always ignored.